spring-cloud-context/pom.xml

```diff
@@ -53,6 +53,10 @@
 			<artifactId>spring-boot-autoconfigure-processor</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
```

Comment:
Ug, maybe it should go the other way? `ProxyUtils` in context. I think it makes more sense here anyway.